src/Lib/Items/CanvasImage.vala

```diff
@@ -103,6 +111,26 @@ public class Akira.Lib.Items.CanvasImage : Goo.CanvasImage, Akira.Lib.Items.Canv
         });
     }
 
+    private bool is_width_small (int width, int height) {
+        // Calculate the width of the image if height was 1 without changing image size
+        // If width is less than 1 then it is 0 we will set it to 1
+        if (width / height < 1) {
+            size.width = 2;
+            return true;
+        }
+        return false;
+    }
+
+    private bool is_height_small (int width, int height) {
+        // Calculate the height of the image if width was 2 without changing image size
+        // If height is less than 1 then it is 0 we will set it to 1
```

Comment:
I'm sorry but these comments are very confusing.
English is not my first language either, but these sentences don't make much sense.